### PR TITLE
Support tests/export to get a text format for archives

### PR DESCRIPTION
**Pull request by coolo**

### CODE REVIEW

File: lib/OpenQA/WebAPI/Controller/Test.pm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2015 SUSE Linux GmbH
+# Copyright (C) 2016 SUSE LLC
```
Comment (aaannz) [inline lib/OpenQA/WebAPI/Controller/Test.pm]: Hohoo. This is a test if I'm paying attention, right? :)

File: lib/OpenQA/WebAPI/Controller/Test.pm [hunk 2]
```diff
@@ -472,5 +472,37 @@ sub add_comment {
...
+	if ($self->param('from')) {
+	    push(@conds, { id => { '>=' => $self->param('from') } });
+	}
+	if ($self->param('to')) {
+	    push(@conds, { id => { '<' => $self->param('to') } });
+	}
+	my $jobs = $group->jobs->search({ -and => \@conds}, { order_by => 'id' } );
+	while (my $job = $jobs->next) {
+	    next if ($job->result eq OpenQA::Schema::Result::Jobs::OBSOLETED);
+	    $self->write_chunk(sprintf("Job %d: %s is %s\n", $job->id, $job->name, $job->result));
```
Comment (aaannz) [inline lib/OpenQA/WebAPI/Controller/Test.pm]: "Job XXX is passed", "Job XXX is failed". Is that correct? Isn't "has" more appropriate (except for 'incomplete' jobs).

**Comment by aaannz** [comment]
Just curious what is the use case. Is this to dump old jobs and then erase them from DB? How about machine parsing old archives, wouldn't something like csv export be better?

File: lib/OpenQA/WebAPI/Controller/Test.pm (hunk 1 above)
Comment (coolo) [inline lib/OpenQA/WebAPI/Controller/Test.pm]: right - as I said before: I don't care what the lawyer says - sue me!

File: lib/OpenQA/WebAPI/Controller/Test.pm (hunk 2 above)
Comment (coolo) [inline lib/OpenQA/WebAPI/Controller/Test.pm]: the correct form is done and passed :)

**Comment by coolo** [comment]
We only want to archive once and we want humans to find information in these archives - so csv is considered too verbose

File: lib/OpenQA/WebAPI/Controller/Test.pm (hunk 1 above)
Comment (okurz) [inline lib/OpenQA/WebAPI/Controller/Test.pm]: @aaannz where is your copyright checker script? Don't waste manual review time with this

File: lib/OpenQA/WebAPI/Controller/Test.pm (hunk 2 above)
Comment (okurz) [inline lib/OpenQA/WebAPI/Controller/Test.pm]: @aaannz not "has" because you don't describe actions here but just the state

**Comment by okurz** [comment]
where is the test?

**Comment by coolo** [comment]
Mr. LongSummary - read before post

File: lib/OpenQA/WebAPI/Controller/Test.pm (hunk 1 above)
Comment (aaannz) [inline lib/OpenQA/WebAPI/Controller/Test.pm]: Too low on priority list :)

Comment (okurz) [inline lib/OpenQA/WebAPI/Controller/Test.pm]: then just ignore the changes in the copyright header

**Comment by coolo** [comment]
Added test now

**Comment by okurz** [comment]
LGTM

**Pull request merged**